conf/messages.en

```diff
@@ -958,6 +958,7 @@ declaration.commodityMeasure.units.hint = Enter weight in kilograms
 
 declaration.summary.locations.header = Locations
 declaration.summary.parties.header = Parties
+supplementary.summary.parties.header = Parties
```

Comment:
What's this new key for? I don't think you are using it.  In general we are trying to get rid of the "supplementary...." message keys.

Reply:
key has been removed